Makefile

```diff
@@ -21,7 +21,7 @@ SHELL := /usr/bin/env bash
 # Docker build config variables
 CREDENTIALS_PATH 			?= /cft/workdir/credentials.json
 DOCKER_ORG 				:= gcr.io/cloud-foundation-cicd
-DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	?= 1.0.1
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	:= 2.1.0
```

Comment:
As per https://github.com/googlecloudplatform/cloud-foundation-toolkit/pull/216, this should be `2.2.0`.

Reply:
Fixed